### DISCUSSION TITLE
Android graphics settings (settings that can actually reduce lag on bad devices)

### DISCUSSION THREAD

**Discussion by Lunaero1** [ppy/osu#19091] [Ideas] [answered]
So I have a question and I do not know if anyone else has asked this type of thing before (yes I've checked and don't see it anywhere) but, couldn't it be possible to optimize the android platform for lower end devices by adding settings that can actually lower the load on the device (such as disabling certain animations, disabling the internal browser inside the game, disabling shaders, etc) to help improve performance on android as a whole for those with lower end devices? I am asking since my own tablet struggles with osu!lazer due to the immense complexity of all the animations and the high load on the GPU and CPU as a whole and it would just be nice to have this type of feature. 
If I am posting this in the wrong place or if it seems really awkward I am sorry, this is my first time trying to ask this type of question.

**Comment by smoogipoo** [comment, marked as answer]
No. Optimisations are an ongoing process and we haven't even started optimising for mobile yet.

https://github.com/ppy/osu/discussions/18929#discussioncomment-3042252
